packages/lavapack/src/runtime.js

```diff
@@ -70,6 +70,17 @@
 })({ onStatsReady }) : () => {}
   const createKernel = // LavaMoat Prelude
 (function () {
+  const globalRef = globalThis;
+
+  const {
+    includeSnow,
+  } = {"includeSnow":false}
+
+  const SNOW = globalRef.SNOW;
+  if (!SNOW && includeSnow) {
```

Comment:
The `includeSnow` constant seems redundant...? (same applies to the other places with the same pattern)

```suggestion
  const { SNOW } = globalRef;
  if (!SNOW) {
```